Apple-TV/VLCServerBrowsingTVViewController.m

```diff
@@ -48,7 +48,7 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
 
-    self.nothingFoundLabel.text = NSLocalizedString(@"FOLDER_EMPTY", nil);
```

Comment:
Isn't that code related to network browsing on the Apple TV?  If so then we still have folders here and this code needs to stay. I also wouldn't change any AppleTV related code in this PR